HOK.Core/HOK.Core/WpfUtilities/FeedbackUI/FeedbackCommand.cs

```diff
@@ -21,10 +21,10 @@ public Result Execute(ExternalCommandData commandData, ref string message, Eleme
 
             try
             {
-                // (Konrad) We are gathering information about the addin use. This allows us to
-                // better maintain the most used plug-ins or discontiue the unused ones.
-                AddinUtilities.PublishAddinLog(
-                    new AddinLog("Feedback Tool", commandData.Application.Application.VersionNumber));
+                //// (Konrad) We are gathering information about the addin use. This allows us to
+                //// better maintain the most used plug-ins or discontiue the unused ones.
+                //AddinUtilities.PublishAddinLog(
+                //    new AddinLog("Feedback Tool", commandData.Application.Application.VersionNumber));
```

Comment:
Just confirming that these changes weren't meant to be temporary.

Reply:
I am still contemplating how best to address this one. Maybe you can toss some ideas my way. Basically the issue is that HOK.Core has code that produced the "Feedback" button on the ribbon. Since its a button on the ribbon I dropped in code for the stats collection so that it sends off a log to MC when user clicks the button. It helps us keep track of what tools are being used and which we can eventually discontinue. However, having the above code in there causes a circular reference. Now we can do one of the following: 
- comment it out and live without the stats for this particular button being collected.
- i can split that specific button code out into its own assembly. removing it from HOK.Core will remove the circular references. 

Any other ideas?